samples/nrf9160/nrf_cloud_rest_agps/Kconfig

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "nRF Cloud REST A-GPS API Sample"
+
+config REST_SAMPLE_NAME
```

Comment:
I just wonder, does it really needs to be configurable? IMO a startup text could be fixed for a sample.

Reply:
This is left over from a bigger (unreleased) sample that supported multiple REST services.  Clearly I missed some cleanup.